website/pages/docs/upgrading/instructions/upgrade-to-1-8-x.mdx

```diff
@@ -0,0 +1,118 @@
+---
+layout: docs
+page_title: Upgrading to 1.8.4
+sidebar_title: Upgrading to 1.8.4
+description: >-
+  Specific versions of Consul may have additional information about the upgrade
+  process beyond the standard flow.
+---
+
+# Upgrading to 1.8.4
+
+## Introduction
+
+This guide explains how to best upgrade a multi-datacenter Consul deployment that is using
+a version of Consul >= 1.6.9 and < 1.8.4 while maintaining replication. If you are on a version
+older than 1.6.9, please follow the link for the version you are on from [here](/docs/upgrading/instructions).
+
+In this guide, we will be using an example with two datacenters (DCs) and will be
+referring to them as DC1 and DC2. DC1 will be the primary datacenter.
+
+## Requirements
+
+- All Consul servers should be on a version of Consul >= 1.6.9 and < 1.8.4.
+
+## Assumptions
+
+This guides makes the following assumptions:
+
+- You have at least two datacenters configured and have ACL replication enabled. If you are
```

Comment:
Is this directly related to [removing the opt-in ACL enforcement](https://www.consul.io/docs/upgrading/upgrade-specific#consul-1-8-0) (and if so, would be helpful to note), or does this guide just follow the same steps as the general upgrade process?

Reply:
Nope, this guide follows the general upgrade process. All upgrade instructions assume you have ACL replication enabled and provide upgrade steps in a fashion that should keep things happy. That's mostly due to the specific steps required when moving pre-1.4.0 to post-1.4.0. There isn't anything special being done due to the opt-in of ACL enforcement there.